core/primitives/Cargo.toml

```diff
@@ -41,6 +38,9 @@ dump_errors_schema = ["near-rpc-error-macro/dump_errors_schema"]
 [dev-dependencies]
 bencher = "0.1.5"
 
+[package.metadata.cargo-udeps.ignore]
+normal = ["jemallocator"]
```

Comment:
what's this for?

Reply:
It is to whitelist the dependency, so it is not marked as not used by cargo-udeps.

Reply:
That I understand. What I meant is why you need to whitelist this dependency.

Reply:
cargo-udeps could not understand that it is used :man_shrugging: